auslib/test/admin/views/test_releases.py

```diff
@@ -735,39 +691,34 @@ def testPostRevisionRollback(self):
             data=dict(
                 data=data,
                 product='d',
-                version='222.0',
                 data_version=1,
             )
         )
         self.assertStatusCode(ret, 200)
 
-        # XXX why does the data_version increment twice?
         data = json.dumps(dict(detailsUrl='blah', fakePartials=False, schema_version=1))
         ret = self._post(
             '/releases/d',
             data=dict(
                 data=data,
                 product='d',
-                version='333.0',
-                data_version=3,
+                data_version=2,
             )
         )
         self.assertStatusCode(ret, 200)
 
         table = dbo.releases
         row, = table.select(where=[table.name == 'd'])
-        self.assertEqual(row['version'], '333.0')
-        self.assertEqual(row['data_version'], 5)
+        self.assertEqual(row['data_version'], 3)
         data = json.loads(row['data'])
         self.assertEqual(data['fakePartials'], False)
 
         query = table.history.t.count()
         count, = query.execute().first()
-        self.assertEqual(count, 2 * 2)
+        self.assertEqual(count, 2)
 
-        # Oh no! We prefer the version 222.0
         row, = table.history.select(
-            where=[table.history.version == '222.0'],
+            where=[table.history.product == 'd'],
             limit=1
         )
```

Comment:
I think we ought to add some sort of reverse sorting here, to ensure we revert the most-recent change.